### ISSUE TITLE
Add GPU details #4

### ISSUE THREAD

**Issue by pierrelefevre** [kthcloud/console#197]
A status of available GPUs may help users to figure out how many GPUs are available before approaching a tier upgrade.

Add a section that displays Free GPUs.
If possible, group by GPU name/type.
Add which instance/VM is using the available GPUs.

**Comment by pierrelefevre** [comment]
Moving https://github.com/kthcloud/visualize/issues/4 here

**Comment by pierrelefevre** [comment]
Considering today's meeting, is this worth implementing, if we're moving all the GPUs into MAIA? @harsha5500

**Comment by harsha5500** [comment]
I agree. We should suspend this for now. We can revisit it if there is a change in the request mechanism.
